bpf/bpf_lxc.c

```
@@ -719,7 +719,11 @@ int handle_ingress(struct __sk_buff *skb)
 
 	case bpf_htons(ETH_P_ARP):
 		ep_tail_call(skb, CILIUM_CALL_ARP);
+#ifndef POLICY_ENFORCEMENT_MODE
```

A couple of notes here:
* If we don't handle ARP for this new mode, then the `ep_tail_call()` call above should also be wrapped inside this ifdef. If we still handle ARP, then I don't understand the motivation behind ignoring errors when attempting to call into the ARP program in this mode.
* The naming is a bit confusing from datapath perspective: "If policy mode is not defined then drop". I think it'd be clearer if this were more specific, something like `#ifdef HANDLE_ARP`.